src/AmyrisBio/utils.fs

```diff
@@ -142,7 +142,7 @@ module utils =
     /// Returns files in dir ending with one of a list of suffices
     let filesEndingWith inDir suffixList =
         Directory.GetFiles(inDir)
-        |> Seq.filter (fun name -> List.exists (fun suffix -> name.EndsWith(suffix)) suffixList)
+        |> Seq.filter (fun name -> List.exists (fun (suffix : string) -> name.EndsWith(suffix)) suffixList)
```

Comment:
please note, string.EndsWith received new overloads in netstandard2.1 making the compiler complain about ambiguity. This type annotation resolves that ambiguity